### PR TITLE
Connector Source for previously certified EXPOCAD connector

### PR DESCRIPTION
**Pull request by mikefactorial**
This PR contains the source for a previously certified connector that was not open sourced at the time. There are several updates to the actions in this source as well including new actions related to finance as well as the required readme.md and metadata in the swagger json.
---
Please check the following conditions for your PR.

- [ x] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [ x] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

### CODE REVIEW

**Comment by sriyen-msft** [comment]
@mikefactorial The swagger is not up-to-date with the swagger you have provided in the certification portal. Please let us know if this is an update to your connector and you want to update your connector.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@mikefactorial The swagger is not up-to-date with the swagger you have provided in the certification portal. Please let us know if this is an update to your connector and you want to update your connector.

**Comment by mikefactorial** [comment]
Hi @sriyen-msft. Thanks for the prompt response! 

This is an update, but also the first time this connector's source has been submitted to the repo since it was previously certified before the requirement for open sourcing. 

Following the instructions in the docs my assumption was we needed to submit the PR with the updates prior to submitting the updates to the connector via the ISV Studio. Let me know if did something out of order. Thanks again!

From the Docs: “Updates to certified connectors must first be made through a pull request to the dev branch for review by the connector owner. Once a pull request has been merged to the dev branch, the connector owner can submit the connector for certification through the Connector certification tab in ISV Studio. Once certified, the Microsoft Certification team will handle merging the updates from dev to master.” - https://github.com/Microsoft/PowerPlatformConnectors#contributing

File: certified-connectors/EXPOCAD/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#e30613",
+    "capabilities": [],
+    "publisher": "Michael Ochs"
```
Comment (sriyen-msft) [inline certified-connectors/EXPOCAD/apiProperties.json:20]: @mikefactorial The `publisher` property value should not be an individual name. Can you please provide the responsible company or org name.

Documentation - [Adding metadata](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#adding-metadata)

File: certified-connectors/EXPOCAD/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  }
```
Comment (sriyen-msft) [inline certified-connectors/EXPOCAD/apiProperties.json:21]: @mikefactorial Please add `stackOwner` property as well

File: certified-connectors/EXPOCAD/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,4316 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v3",
+    "title": "EXPOCAD",
+    "description": "This service interfaces with live EXPOCAD\u00ae event data.  Please use PUT, POST, and DELETE operations with discretion. Keep authorization codes private.  If your code has been compromised, contact tech@expocad.com immediately.",
```
Comment (sriyen-msft) [inline certified-connectors/EXPOCAD/apiDefinition.swagger.json:6]: @mikefactorial Unicodes are not allowed, Can you please replace unicodes with respective English characters

File: certified-connectors/EXPOCAD/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,4316 @@
...
+    "contact": {
+      "name": "EXPOCAD Developer Support",
+      "email": "developer@expocad.com"
```
Comment (sriyen-msft) [inline certified-connectors/EXPOCAD/apiDefinition.swagger.json:9]: @mikefactorial Please add `website` as well in `contact` section

Documentation - [Adding Metadata](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#adding-metadata)

File: certified-connectors/EXPOCAD/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,4316 @@
...
+        "description": "Unrents the specified booth in the supplied event."
+      }
+    },
+    "/{ClientName}/booths/hold": {
+      "put": {
+        "tags": [
+          "Booths"
+        ],
+        "summary": "Holds the specified booth",
+        "description": "EXPOCAD allows for a booth to be placed on hold to either an existing Exhibitor (by Id) or to \r\n            a temporary Exhibitor (by Name).<br /> Choose one parameter or the other -  'ExhibitorId' or 'ExhibitorName'.<br /><br />\r\n            If both paramerters are supplied, the hold will default to the Exhibitor ID.",
```
Comment (sriyen-msft) [inline certified-connectors/EXPOCAD/apiDefinition.swagger.json:356]: @mikefactorial Please remove the extra spacing in the value

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/EXPOCAD/apiProperties.json (hunk 1 above)
Comment (mikefactorial) [reply certified-connectors/EXPOCAD/apiProperties.json]: Hi @sriyen-msft this has been resolved in the latest commit

File: certified-connectors/EXPOCAD/apiProperties.json (hunk 2 above)
Comment (mikefactorial) [reply certified-connectors/EXPOCAD/apiProperties.json:22]: Hi @sriyen-msft this has been resolved in the latest commit

File: certified-connectors/EXPOCAD/apiDefinition.swagger.json (hunk 4 above)
Comment (mikefactorial) [reply certified-connectors/EXPOCAD/apiDefinition.swagger.json:10]: Added "url" per the documentation.

File: certified-connectors/EXPOCAD/apiDefinition.swagger.json (hunk 3 above)
Comment (mikefactorial) [reply certified-connectors/EXPOCAD/apiDefinition.swagger.json]: Hi @sriyen-msft this has been resolved in the latest commit

File: certified-connectors/EXPOCAD/apiDefinition.swagger.json (hunk 5 above)
Comment (mikefactorial) [reply certified-connectors/EXPOCAD/apiDefinition.swagger.json]: This has been resolved in the latest commit

**Comment by mikefactorial** [comment]
@sriyen-msft I've made the requested changes. I wasn't sure if I needed to rerequest your review after making the changes, but I did so if that was a mistake in protocol let me know so I don't do it again next time :)

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**